templates/go-audit.yaml.erb

```diff
@@ -87,21 +88,76 @@ log:
   # LUTC          = 32 // if Ldate or Ltime is set, use UTC rather than the local time zone
   #
   # See also: https://golang.org/pkg/log/#pkg-constants
-  flags: <%= scope['::goaudit::log_flags'] %>
-
-rules:
-  # Watch all 64 bit program executions
-  - -a exit,always -F arch=b64 -S execve
-  # Watch all 32 bit program executions
-  - -a exit,always -F arch=b32 -S execve
-  # Enable kernel auditing (required if not done via the "audit" kernel boot parameter)
-  # You can also use this to lock the rules. Locking requires a reboot to modify the ruleset.
-  # This should be the last rule in the chain.
-  - -e 1
```

Comment:
There doesn't appear to be any handling that ensures this is the last rule in the set.